index.html

```diff
@@ -880,30 +886,40 @@
             In a <a>controlling user agent</a>, the <dfn for=
             "Presentation"><code>defaultRequest</code></dfn> attribute MUST
             return the <a>default presentation request</a> if any,
-            <code>null</code> otherwise. In a <a>receiving browsing
-            context</a>, it MUST return <code>null</code>.
+            <code>null</code> otherwise. On setting, the <a>default
+            presentation request</a> MUST be set to the new value.
           </p>
           <p>
-            If set by the <a>controller</a>, the value of the <a for=
-            "Presentation">defaultRequest</a> attribute SHOULD be used by the
-            <a>controlling user agent</a> as the <dfn>default presentation
-            request</dfn> for that <a>controlling browsing context</a>. If the
-            the document object's <a>active sandboxing flag set</a> has the
-            <a>sandboxed presentation browsing context flag</a> set, the
-            <a>controlling user agent</a> SHOULD consider the <a>default
-            presentation request</a> for that browsing context to be
-            <code>null</code>. When the <a>controlling user agent</a> wishes to
-            initiate a <a>PresentationConnection</a> on the behalf of that
-            browsing context, it MUST <a>start a presentation</a> using the
-            <a>default presentation request</a> for the <a>controller</a> (as
-            if the controller had called <code>defaultRequest.start()</code>).
+            In a user agent that is a <a>receiving user agent</a> but not a
```

Comment:
This sounds better and reflects the intent described below.  Note that in this case it's unlikely that the PresentationRequest constructor is implemented at all, in which case there would be no way to set it anyway.